docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+    def hello():
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    ```
+
+Since we used powertools api_gateway event handler we don't need to know internal event structure. All we do we instantiate `ApiGatewayResolver` class and we call it in our `lambda_handler`.
+Since we added route annotation as a decorator for our methods we can directly use parameters passed in request.
+In order for SAM to build our package correctly we specified lambda powertools package in our requirements file. 
+It is then automatically downloaded by SAM and added to lambda zip file. 
+
+!!! tip
+    If you are interested in learning how python decorators work under the hood you can follow great article from [Real Python](https://realpython.com/primer-on-python-decorators/).
+## Structured Logging
+In next step you have been tasked to propose production grade logging capabilities to your lambda code.
+You decided to follow [structured logging approach](https://docs.aws.amazon.com/lambda/latest/operatorguide/parse-logs.html) which means that you don’t write hard-to-parse and hard-to-keep-consistent prose in your logs but that you log events that happen in a context instead. 
+We want our log event to be in JSON format. This will allow us to search through it easily in CloudWatch Logs or Cloudwatch Insight.
+Initial idea might be to initiate logger from standard library and use `pythonjsonlogger` library to simplify structured log creation.
+
+=== "app.py"
+
+    ```python hl_lines="1 4 8-13 20 26 31"
+    import logging
+    import os
+
+    from pythonjsonlogger import jsonlogger
+
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+    logger = logging.getLogger("APP")
+    logHandler = logging.StreamHandler()
+    formatter = jsonlogger.JsonFormatter(fmt="%(asctime)s %(levelname)s %(name)s %(message)s")
+    logHandler.setFormatter(formatter)
+    logger.addHandler(logHandler)
+    logger.setLevel(os.getenv("LOG_LEVEL", "INFO"))
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        logger.debug(event)
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    python-json-logger
+    ```
+So in our first attempt we create new APP Logger. We configure Log Handler, formatter and log level. Then we use this logger to log required information. We will see logs structured like that
```

Comment:
If you want to highlight the complexity because of number of steps, you can have an enumeration: 
* create  app logger
* configure handler and formatter
* set log level

Just a thought, it doesn't have to be a list, can also be in one sentence.

Reply:
Enumeration looks good, fixing